plugin/util/parse_bindings.go

```diff
@@ -88,12 +88,15 @@ func parseBindingObjList(topList *ast.ObjectList) (map[string]StringSet, error)
 
 		resourceList := item.Val.(*ast.ObjectType).List
 		for _, rolesItem := range resourceList.Items {
-			key := rolesItem.Keys[0].Token.Text
-			switch key {
+			if len(rolesItem.Keys) == 0 || rolesItem.Keys[0].Token.Value() == nil {
+				merr = multierror.Append(merr, fmt.Errorf("invalid nil item under key %q in resource %q (line %d)", key, resourceName, rolesItem.Assign.Line))
+			}
+			roleKey := rolesItem.Keys[0].Token.Value().(string)
```

Comment:
Not sure if `Value()` can be anything other than a string, but I would feel more comfortable if we did:

```golang
roleKey, ok := rolesItem.Keys[0].Token.Value().(string)
if !ok {
  // handle error
}
```

Reply:
Yeah that's exactly what I asked for too :-)

Reply:
my bad poor reading comprehension